sql/src/main/java/io/crate/executor/transport/AlterTableOperation.java

```diff
@@ -211,14 +269,17 @@ public void fail(@Nonnull Throwable t) {
         // merge settings
         Settings.Builder settingsBuilder = Settings.builder();
         settingsBuilder.put(indexTemplateMetaData.settings());
-        settingsBuilder.put(newSettings);
+        if (newSettings != null) {
```

Comment:
It's not, I did't see that I could pass Settings.Empty into the function, my mistake.